README.md

```diff
@@ -5,3 +5,24 @@ implementation, and reference Knative client libraries.
 
 If you are interested in contributing, see the Knative community [contribution
 guide](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md).
+
+
+## How to build
+
+Knative Client uses [go mod](https://github.com/golang/go/wiki/Modules#quick-start) for dependencies and thus requires Go `1.11`+ to build.
+
+**Building:**
+
+```sh
+$ go build ./cmd/...
```

Comment:
Why not just say?
`$ go build ./cmd/kn/main.go`
since that is what you actually have to do

Reply:
btw, thanks this works great!

Reply:
Because Go will automatically build and test all packages in the root folder with that command: https://github.com/golang/go/wiki/Modules#daily-workflow

But yeah, `./cmd/kn/main.go` works as well!

Reply:
Ha, I thought you were just too lazy and wrote `...` (like I do sometimes)

Reply:
My fingers have `go build ./cmd/kn`. There are lots of options!